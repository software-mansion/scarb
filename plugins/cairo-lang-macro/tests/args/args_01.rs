use cairo_lang_macro::{attribute_macro, ProcMacroResult, TokenStream};

#[attribute_macro(unsupported_key = "some::path")]
fn t1(_a: TokenStream, _b: TokenStream) -> ProcMacroResult {
    ProcMacroResult::new(TokenStream::empty())
}

fn main() {}
