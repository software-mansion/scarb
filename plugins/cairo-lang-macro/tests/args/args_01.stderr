error: only `parent` argument is supported
 --> tests/args/args_01.rs:3:35
  |
3 | #[attribute_macro(unsupported_key = "some::path")]
  |                                   ^

warning: unused imports: `ProcMacroResult` and `TokenStream`
 --> tests/args/args_01.rs:1:41
  |
1 | use cairo_lang_macro::{attribute_macro, ProcMacroResult, TokenStream};
  |                                         ^^^^^^^^^^^^^^^  ^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
